### ISSUE TITLE
Error Code without number after trying to connect/Unable to get a gamesync ID

### ISSUE THREAD

**Issue by MegaKBang** [kuroppoi/entralinked#19]
I've set everything up as I was instructed in the quick setup guide and was able to fix the 60000 error by switching to my mobile hotspot as advised in the troubleshooting guide.
However now I get a new error that doesn't have a number like the other ones.
The error is in german because I use a german copy, but basically it says that the connection got interrupted and wants me to check out the nintendo support site. I've tried both the gamesync settings at the title screen and gamesync at the C-Gear. The error appears after the purple screen where it says "gamesync"(in german) appears and loads for a bit, trying to connect to the DNS serverat the top.
Extra info that may be relevant: 
-I am using a german copy of black on a European n3DS.
-I am using the mobile hotspot from my android phone.
-other DNS-exploits like riiconnect have been working for getting the mystery gifts
-I am unable to even get a gamesync ID as when I start up the settings, I get the same error after it tries to create the necessary game data for gamesync.(I am using a completely fresh save file with 2 badges and this game cartrigde never connected to the internet before except for mystery gift).

I am sorry, if some bits sound confusing, english isn't my main language.

At last, here is the log from entralink:

2023-09-18 19:15:00.259  INFO : Loading configuration ...
2023-09-18 19:15:00.270  INFO : No configuration file exists - default configuration will be used
2023-09-18 19:15:00.328  INFO : Using configuration Configuration[hostName=local, clearPlayerDreamInfoOnWake=true, allowOverwritingPlayerDreamInfo=false, allowPlayerGameVersionMismatch=false, allowWfcRegistrationThroughLogin=true]
2023-09-18 19:15:00.436  INFO : Loading DLC ...
2023-09-18 19:15:00.437  INFO : Extracting default DLC files ...
2023-09-18 19:15:00.578  INFO : Loaded 59 DLC file(s)
2023-09-18 19:15:00.580  INFO : Loading user and profile data ...
2023-09-18 19:15:00.582  INFO : Loading player data ...
2023-09-18 19:15:00.770  INFO : Mapping GameSpy request types ...
2023-09-18 19:15:01.080  INFO : Creating certificate keystore ...
2023-09-18 19:15:01.096  INFO : No keystore found - generating one!
2023-09-18 19:15:01.756  INFO : Loading C-Gear and Pokédex skin previews ...
2023-09-18 19:15:02.022  INFO : Cached 57 skin previews
2023-09-18 19:15:02.026  INFO : Starting servers ...
2023-09-18 19:15:02.028  INFO : Starting HTTP server ...
2023-09-18 19:15:02.030  INFO : Starting Javalin ...
2023-09-18 19:15:02.056  INFO : jetty-11.0.15; built: 2023-04-11T18:37:53.775Z; git: 5bc5e562c8d05c5862505aebe5cf83a61bdbcb96; jvm 17.0.8+9-LTS-211
2023-09-18 19:15:02.107  INFO : Session workerName=node0
2023-09-18 19:15:02.117  INFO : Started i.j.j.@68b3248a{/,null,AVAILABLE}
2023-09-18 19:15:02.138  INFO : Started ServerConnector@59a09be{HTTP/1.1, (http/1.1)}{0.0.0.0:80}
2023-09-18 19:15:02.141  INFO : x509=X509@7de6549d(server,h=[],a=[],w=[*.*]) for Server@12214f2f[provider=null,keyStore=null,trustStore=null]
2023-09-18 19:15:02.203  WARN : Protocol SSLv3 not excluded for Server@12214f2f[provider=null,keyStore=null,trustStore=null]
2023-09-18 19:15:02.204  WARN : Weak cipher suite SSL_RSA_WITH_RC4_128_SHA enabled for Server@12214f2f[provider=null,keyStore=null,trustStore=null]
2023-09-18 19:15:02.205  WARN : Weak cipher suite SSL_RSA_WITH_RC4_128_SHA enabled for Server@12214f2f[provider=null,keyStore=null,trustStore=null]
2023-09-18 19:15:02.206  WARN : Weak cipher suite SSL_RSA_WITH_RC4_128_MD5 enabled for Server@12214f2f[provider=null,keyStore=null,trustStore=null]
2023-09-18 19:15:02.206  WARN : Weak cipher suite SSL_RSA_WITH_RC4_128_MD5 enabled for Server@12214f2f[provider=null,keyStore=null,trustStore=null]
2023-09-18 19:15:02.224  INFO : Started ServerConnector@750cd36d{SSL, (ssl, http/1.1)}{0.0.0.0:443}
2023-09-18 19:15:02.226  INFO : Started Server@4492b393{STARTING}[11.0.15,sto=0] @3148ms
2023-09-18 19:15:02.227  INFO : 
       __                  ___          ______
      / /___ __   ______ _/ (_)___     / ____/
 __  / / __ `/ | / / __ `/ / / __ \   /___ \
/ /_/ / /_/ /| |/ / /_/ / / / / / /  ____/ /
\____/\__,_/ |___/\__,_/_/_/_/ /_/  /_____/

       https://javalin.io/documentation

2023-09-18 19:15:02.244  INFO : Static file handler added: StaticFileConfig(hostedPath=/dashboard, directory=/dashboard, location=CLASSPATH, precompress=false, aliasCheck=null, headers={Cache-Control=max-age=0}, skipFileFunction=Function1<jakarta.servlet.http.HttpServletRequest, java.lang.Boolean>, mimeTypes=io.javalin.http.staticfiles.MimeTypesConfig@7fe87c0e). File system location: 'jar:file:/C:/Users/kbang/Downloads/Mod%20Folder%20V3.1/entralinked/entralinked.jar!/dashboard/'
2023-09-18 19:15:02.246  INFO : Static file handler added: StaticFileConfig(hostedPath=/sprites, directory=/sprites, location=CLASSPATH, precompress=false, aliasCheck=null, headers={Cache-Control=max-age=0}, skipFileFunction=Function1<jakarta.servlet.http.HttpServletRequest, java.lang.Boolean>, mimeTypes=io.javalin.http.staticfiles.MimeTypesConfig@478530cb). File system location: 'jar:file:/C:/Users/kbang/Downloads/Mod%20Folder%20V3.1/entralinked/entralinked.jar!/sprites/'
2023-09-18 19:15:02.251  INFO : Listening on http://localhost:80/
2023-09-18 19:15:02.253  INFO : Listening on https://localhost:443/
2023-09-18 19:15:02.265  INFO : You are running Javalin 5.5.0 (released May 1, 2023. Your Javalin version is 139 days old. Consider checking for a newer version.).
2023-09-18 19:15:02.266  INFO : Javalin started in 238ms \o/
2023-09-18 19:15:02.268  INFO : Staring GameSpy server ...
2023-09-18 19:15:02.348  INFO : GameSpy server listening @ port 29900
2023-09-18 19:15:02.348  INFO : Staring DNS server ...
2023-09-18 19:15:02.362  INFO : DNS server listening @ port 53
2023-09-18 19:15:02.363  INFO : Startup complete! Took a total of 2714 milliseconds

**Comment by kuroppoi** [comment]
Well, it certainly is a bit strange.
If you switched networks to a mobile hotspot, make sure that you restart Entralinked and reconfigure the DNS settings.
Both your DS and computer must be connected to the hotspot.
If possible, I'd like to see a video or screen recording to maybe see what goes wrong.

**Comment by MegaKBang** [comment]
Thanks for the reply.
I double checked both of these points when I tried this before I made this post.
I'll try to upload a recording.

**Comment by kuroppoi** [comment]
If you've restarted Entralinked prior to creating this issue then make sure you attach the correct log as well, as the current log appears to be from first-time startup.

**Comment by MegaKBang** [comment]
Here is the link to the video: https://youtu.be/7iSVgpNv1e4

Also here is the new log:

2023-09-18 20:11:45.358  INFO : Loading configuration ...
2023-09-18 20:11:45.451  INFO : Using configuration Configuration[hostName=local, clearPlayerDreamInfoOnWake=true, allowOverwritingPlayerDreamInfo=false, allowPlayerGameVersionMismatch=false, allowWfcRegistrationThroughLogin=true]
2023-09-18 20:11:45.601  INFO : Loading DLC ...
2023-09-18 20:11:45.639  INFO : Loaded 59 DLC file(s)
2023-09-18 20:11:45.640  INFO : Loading user and profile data ...
2023-09-18 20:11:45.642  INFO : Loading player data ...
2023-09-18 20:11:45.815  INFO : Mapping GameSpy request types ...
2023-09-18 20:11:46.100  INFO : Creating certificate keystore ...
2023-09-18 20:11:46.115  INFO : Cached keystore found - loading it!
2023-09-18 20:11:46.655  INFO : Loading C-Gear and Pokédex skin previews ...
2023-09-18 20:11:46.925  INFO : Cached 57 skin previews
2023-09-18 20:11:46.930  INFO : Starting servers ...
2023-09-18 20:11:46.931  INFO : Starting HTTP server ...
2023-09-18 20:11:46.934  INFO : Starting Javalin ...
2023-09-18 20:11:46.978  INFO : jetty-11.0.15; built: 2023-04-11T18:37:53.775Z; git: 5bc5e562c8d05c5862505aebe5cf83a61bdbcb96; jvm 17.0.8+9-LTS-211
2023-09-18 20:11:47.032  INFO : Session workerName=node0
2023-09-18 20:11:47.046  INFO : Started i.j.j.@2c77344f{/,null,AVAILABLE}
2023-09-18 20:11:47.069  INFO : Started ServerConnector@3dea226b{HTTP/1.1, (http/1.1)}{0.0.0.0:80}
2023-09-18 20:11:47.074  INFO : x509=X509@6723cce7(server,h=[],a=[],w=[*.*]) for Server@74cd798f[provider=null,keyStore=null,trustStore=null]
2023-09-18 20:11:47.137  WARN : Protocol SSLv3 not excluded for Server@74cd798f[provider=null,keyStore=null,trustStore=null]
2023-09-18 20:11:47.138  WARN : Weak cipher suite SSL_RSA_WITH_RC4_128_SHA enabled for Server@74cd798f[provider=null,keyStore=null,trustStore=null]
2023-09-18 20:11:47.139  WARN : Weak cipher suite SSL_RSA_WITH_RC4_128_SHA enabled for Server@74cd798f[provider=null,keyStore=null,trustStore=null]
2023-09-18 20:11:47.140  WARN : Weak cipher suite SSL_RSA_WITH_RC4_128_MD5 enabled for Server@74cd798f[provider=null,keyStore=null,trustStore=null]
2023-09-18 20:11:47.141  WARN : Weak cipher suite SSL_RSA_WITH_RC4_128_MD5 enabled for Server@74cd798f[provider=null,keyStore=null,trustStore=null]
2023-09-18 20:11:47.164  INFO : Started ServerConnector@3c28181b{SSL, (ssl, http/1.1)}{0.0.0.0:443}
2023-09-18 20:11:47.168  INFO : Started Server@7910f355{STARTING}[11.0.15,sto=0] @3347ms
2023-09-18 20:11:47.170  INFO : 
       __                  ___          ______
      / /___ __   ______ _/ (_)___     / ____/
 __  / / __ `/ | / / __ `/ / / __ \   /___ \
/ /_/ / /_/ /| |/ / /_/ / / / / / /  ____/ /
\____/\__,_/ |___/\__,_/_/_/_/ /_/  /_____/

       https://javalin.io/documentation

2023-09-18 20:11:47.189  INFO : Static file handler added: StaticFileConfig(hostedPath=/dashboard, directory=/dashboard, location=CLASSPATH, precompress=false, aliasCheck=null, headers={Cache-Control=max-age=0}, skipFileFunction=Function1<jakarta.servlet.http.HttpServletRequest, java.lang.Boolean>, mimeTypes=io.javalin.http.staticfiles.MimeTypesConfig@28b5662d). File system location: 'jar:file:/C:/Users/kbang/Downloads/Mod%20Folder%20V3.1/entralinked/entralinked.jar!/dashboard/'
2023-09-18 20:11:47.191  INFO : Static file handler added: StaticFileConfig(hostedPath=/sprites, directory=/sprites, location=CLASSPATH, precompress=false, aliasCheck=null, headers={Cache-Control=max-age=0}, skipFileFunction=Function1<jakarta.servlet.http.HttpServletRequest, java.lang.Boolean>, mimeTypes=io.javalin.http.staticfiles.MimeTypesConfig@3c544c9). File system location: 'jar:file:/C:/Users/kbang/Downloads/Mod%20Folder%20V3.1/entralinked/entralinked.jar!/sprites/'
2023-09-18 20:11:47.195  INFO : Listening on http://localhost:80/
2023-09-18 20:11:47.196  INFO : Listening on https://localhost:443/
2023-09-18 20:11:47.208  INFO : You are running Javalin 5.5.0 (released May 1, 2023. Your Javalin version is 139 days old. Consider checking for a newer version.).
2023-09-18 20:11:47.209  INFO : Javalin started in 277ms \o/
2023-09-18 20:11:47.211  INFO : Staring GameSpy server ...
2023-09-18 20:11:47.296  INFO : GameSpy server listening @ port 29900
2023-09-18 20:11:47.296  INFO : Staring DNS server ...
2023-09-18 20:11:47.311  INFO : DNS server listening @ port 53
2023-09-18 20:11:47.312  INFO : Startup complete! Took a total of 2981 milliseconds

**Comment by kuroppoi** [comment]
Well that's unusual.. is this log from _after_ you tried to connect?

**Comment by MegaKBang** [comment]
Yep. Thats the log after I made that video.

**Comment by kuroppoi** [comment]
Quite the head-scratcher, this one.. It appears to just break the connection halfway through but I have no idea why.
Are there any other active connections configured on your DS? What happens if you try to use your home Wi-Fi network instead?

**Comment by MegaKBang** [comment]
When I use my home Wi-Fi I just get the 60000-52210 error thats mentioned in your troubleshooting section.

**Comment by MegaKBang** [comment]
Also there was one more connection I had saved there, but it was from my GF's old Wi-Fi. I deleted that connection and still no change sadly.

**Comment by kuroppoi** [comment]
That's interesting. You mentioned earlier that switching networks fixed error 60000, when the fix you should do is completely different:
https://github.com/kuroppoi/entralinked/wiki/Troubleshooting#60000-followed-by-52200

**Comment by MegaKBang** [comment]
Oh yeah. I guess I somehow mixed up the headers, when I read the guide. I'll try te PID fix out and report back.

**Comment by MegaKBang** [comment]
Alright so there are 2 new things.

1. I switched over to my old OG NDS, and when I use here my phone Wi-Fi it also displays the 60000 error. So it seems that my n3DS doesn't seem to like my phone Wi-Fi, when doing DNS stuff for some reason. So most likely has nothing to do with entralinked I think.
2. I did the PID fix and now I think it works. I finally got my gamesync ID and the game now tells me to wait for a day until I can fully use the feature. I'll try tomorrow again to tuck in a Pokemon and report back here, but I think everything is working now. So thanks in advance for your help and your quick respones!
